packages/host/src/components/AppDrawer/AppDrawer.tsx

```diff
@@ -193,11 +193,11 @@ export const AppDrawer: React.FC = () => {
             icon={<ToolsIcon fontSize="small" color="primary" />}
             text={t('tools')}
           /> */}
-          <AppNavLink
+          {/* <AppNavLink
```

Comment:
thought that we won't get a report implementation in before v1 and this is just hanging around.
Should have probably removed the route as well 🤷